server/src/main/scala/net/wiringbits/models/SiteKey.scala

```diff
@@ -0,0 +1,13 @@
+package net.wiringbits.models
+
+import com.typesafe.config.Config
+import play.api.ConfigLoader
+
+case class SiteKey(string: String) extends SecretValue(string)
```

Comment:
Let's rename to `RecaptchaSiteKey`, this is not a secrent, hence, `extends SecretValue(string)` is unnecessary